py/newsletter.py

```diff
@@ -19,8 +19,8 @@
 
 # create the document body
 with doc:
-    # enclose all of the elements within a table cell.
-    with table().add(tbody()).add(tr()).add(td()):
+    # enclose all of the elements within div elements (currently set to ef-styling)
+    with div(cls='ef-wrapper').add(div(cls='ef-container')).add(div(cls='ef-table ef-outer')):
```

Comment:
So why aren't we wrapping everything in a table anymore? I thought that was how we resolved the background issue. Does a div work the same?

Reply:
Using the divs as the outer layers should work the same. This is also how we've been doing it for the html files.